SPECS/pytorch/pytorch.spec

```
@@ -0,0 +1,113 @@
+%global _empty_manifest_terminate_build 0
+Summary:        Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+Name:           pytorch
+Version:        1.13.1
+Release:        1%{?dist}
+License:        BSD-3-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://pytorch.org/
+Source0:        https://github.com/pytorch/pytorch/releases/download/v%{version}/pytorch-v%{version}.tar.gz#/pytorch-%{version}.tar.gz
+Source1:        %{name}-%{version}-submodules.tar.gz
+
+%description
+PyTorch is a Python package that provides two high-level features:
+- Tensor computation (like NumPy) with strong GPU acceleration
+- Deep neural networks built on a tape-based autograd system
+You can reuse your favorite Python packages such as NumPy, SciPy and Cython to extend PyTorch when needed.
+
+%package -n     python3-pytorch
+Summary:        Tensors and Dynamic neural networks in Python with strong GPU acceleration.
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  python3-PyYAML
+BuildRequires:  python3-devel
+BuildRequires:  python3-filelock
+BuildRequires:  python3-hypothesis
+BuildRequires:  python3-jinja2
+BuildRequires:  python3-numpy
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-typing-extensions
+Requires:       python3-filelock
+Requires:       python3-numpy
+Requires:       python3-typing-extensions
+Provides:       python-torch
+
+%description -n python3-pytorch
+PyTorch is a Python package that provides two high-level features:
+- Tensor computation (like NumPy) with strong GPU acceleration
+- Deep neural networks built on a tape-based autograd system
+You can reuse your favorite Python packages such as NumPy, SciPy and Cython to extend PyTorch when needed.
+
+%package help
+Summary:        Development documents and examples for torch
+Provides:       python3-pytorch-doc
+
+%description help
+PyTorch is a Python package that provides two high-level features:
+- Tensor computation (like NumPy) with strong GPU acceleration
+- Deep neural networks built on a tape-based autograd system
+You can reuse your favorite Python packages such as NumPy, SciPy and Cython to extend PyTorch when needed.
+
+%prep
+%setup -q
+%setup -q -T -D -a 1
+
+%build
+export USE_CUDA=0
+export BUILD_CAFFE2=0
+%py3_build
+
+%install
+%py3_install
+install -d -m755 %{buildroot}/%{_pkgdocdir}
+if [ -d doc ]; then cp -arf doc %{buildroot}/%{_pkgdocdir}; fi
+if [ -d docs ]; then cp -arf docs %{buildroot}/%{_pkgdocdir}; fi
+if [ -d example ]; then cp -arf example %{buildroot}/%{_pkgdocdir}; fi
+if [ -d examples ]; then cp -arf examples %{buildroot}/%{_pkgdocdir}; fi
```

Suggestion: let's combine these lines to make it easier to see they do the same thing for different directories:
```suggestion
for directory doc docs example examples; do
  if [ -d $directory ]; then
    cp -arf $directory %{buildroot}/%{_pkgdocdir}
  fi
done
```